tasks/2016/interactive-bots.md

```diff
@@ -9,26 +9,25 @@
 * You need to know how to create a GitHub pull request. Check out the
   [Learn how to create a GitHub Pull Request](https://codein.withgoogle.com/tasks/6541581402243072/)
   task if you aren't sure how to do this, or read through the task
-  description
-  [here](https://github.com/zulip/zulip-gci/blob/master/tasks/submit-a-pull-request.md).
+  description [here](https://github.com/zulip/zulip-gci/tree/master/submit-a-pull-request).
 
 * Update your working copy of Zulip and then create a feature branch. [Learn
   how](../before-every-task.md).
 
 ## Background
 
-Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
+[Zulip bots](https://github.com/zulip/python-zulip-api/tree/master/zulip_bots)
```

Comment:
Probably want to change this whole paragraph to give a very brief overview over zulip bots. I think  `boilerplate` doesn't hit the target 100% here (next line).